rpcd/playbooks/roles/beaver/tasks/beaver_install.yml

```diff
@@ -50,6 +50,9 @@
     - beaver-install
     - beaver-init-file
 
+- name: Create runlevel links
+  command: update-rc.d -f beaver defaults
+
```

Comment:
@mancdaz part of the problem was 'update-rc.d -f beaver enable 2 3 4 5' was failing and this was the most direct solution I found. I did not know you could do something like you pointed out and will test that also.